src/app/services/xml-parsers/generic-parser.service.ts

```diff
@@ -60,6 +61,18 @@ export class GenericParserService {
     return text;
   }
 
+  public parseParagrah(xml: XMLElement): ParagraphData {
+    const paragraphComponent: ParagraphData = {
+      type: 'ParagraphComponent',
+      class: xml.tagName ? xml.tagName.toLowerCase() : '',
```

Comment:
Why this condition? A XMLElement has always a tagName

Reply:
Copy-Paste error... thinking about it, does it actually make sense having a class attribute here?

Reply:
No, in this case having the class `p` for the paragraph component is no actually useful. You can remove it!